presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -115,8 +115,8 @@ public Integer call()
     {
         private static final String DEFAULT_VALUE = "(default: ${DEFAULT-VALUE})";
 
-        @Option(names = "--suite", paramLabel = "<suite>", description = "Name of the suite to run", required = true)
-        public String suite;
+        @Option(names = "--suite", paramLabel = "<suite>", description = "Name of the suite(s) to run (comma separated)", required = true, split = ",")
+        public List<String> suites;
```

Comment:
what happens when i `--suite ""` or `--suite my-suite,` or `--suite my-suite,,my-another-suite`
?

Reply:
Picocli will handle `--suite suite-1 --suite suite-2`. In case of `--suite suite-1,,suite-2` it will create list with an empty string that will be handled by our logic: `java.lang.RuntimeException: java.lang.IllegalArgumentException: No suite with name ''.`